utils/build_grimoirelab

```diff
@@ -80,7 +80,6 @@ def parse_args ():
 
     args_actions = parser.add_argument_group("Actions", "Actions to perform")
     args_actions.add_argument("--build", action='store_true',
-                            default=True,
                             help="Build modules. It is the default action.")
```

Comment:
"Build modules. It is the default action."

The last "It is ..." must be removed, no?